common/apiKey.go

```diff
@@ -7,14 +7,16 @@ import (
 // NewApiKeyHeaderAuthHTTPClient returns a new http client, with automatic API key
 // authentication. Specifically this means that the client will automatically
 // add the API key (as a header) to every request.
+// API key must be in the correct format. This sometimes means adding prefix to the raw Key.
+// TODO maybe this method should add PREFIX argument?
```

Comment:
@eberle1080 I am using this method from paramsbuilder.
The ApiKey argument is in fact already formated ApiKey value. Becuase of string data type it is not explicit.

What do you think about arg rename?
What if we instead we add another argument called prefix? (optional prefix + apikey -> headerValueApiKey)

Reply:
I'd prefer just `headerValue` - that's pretty explicit about what's being passed in. Plus it matches `headerName` in terseness.